streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregate.java

```diff
@@ -32,7 +33,7 @@
 import java.util.Map;
 
 public class KStreamWindowAggregate<K, V, T, W extends Window> implements KStreamAggProcessorSupplier<K, Windowed<K>, V, T> {
-    private static final Logger LOG = LoggerFactory.getLogger(KStreamWindowAggregate.class);
+    private final Logger log = LoggerFactory.getLogger(getClass());
```

Comment:
Necessary, so that the `KStreamWindowReduce` logger will report its own class name instead of `..Aggregate`.